packages/data-helper/src/hooks/useNodeInfo/useNodeInfo.gql-queries.js

```diff
@@ -0,0 +1,248 @@
+import gql from 'graphql-tag';
+import {PredefinedFragments, replaceFragmentsInDocument} from '../../fragments';
+
+const NodeInfoQuery = gql`
+    query NodeInfoQuery($path:String!) {
+        jcr {
+            nodeByPath(path:$path) {
+                name
+                ...node
+                ...NodeCacheRequiredFields
+            }
+        }
+    }
+    ${PredefinedFragments.nodeCacheRequiredFields.gql}
+`;
+
+export class NodeInfoQueryHandler {
```

Comment:
What is the purpose of this class ?

Reply:
none, it's a copy of ActionsRequirementQueryHandler